PicPi/FullImageViewController.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  FullImageViewController.swift
+//  PicPi
+//
+//  Created by Mustafa T. Mohammed on 1/2/21.
+//
+
+import UIKit
+import Promises
+
+class FullImageViewController: UIViewController {
+  private let deleteButton = customBtnRoundCornerBlueWithShadow(type: .custom)
+  private var margins: UILayoutGuide!
+  private var photo: Photo
+  
+  init(photo: Photo) {
+    self.photo = photo
+    super.init(nibName: nil, bundle: nil)
+  }
+  
+  @available(*, unavailable)
+  required init?(coder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    margins = view.layoutMarginsGuide
+    view.backgroundColor = .white
+    title = photo.filename
+    
+    setupDeleteButton()
+    setupImageView()
+  }
+  
+  private func setupImageView() {
+    let imageView = UIImageView()
+    imageView.isUserInteractionEnabled = true
```

Comment:
Is this needed? I've only seen this added to an image view if we wanted to listen for taps

Reply:
forgot what I was testing lol